src/content/docs/design/design-with-code/connect-repo.mdx

```diff
@@ -0,0 +1,22 @@
+---
+page_id: 261fb4ed-83e5-4f92-8f52-8da22749a92e
+title: Connect your repo for custom pages
+sidebar:
+  order: 2
+relatedArticles:
+  - 66f83c18-d261-48b8-a517-648ed1b7b064
+  - b42663d8-5e2a-42cd-859a-f9737a1495df
+  - 743d264d-ef08-4f2e-a42c-2b971152dccf
+---
+
+To enable your design code to override Kinde page design, you need to set up and connect a git repo for the design files. You must also be using a [custom domain](/build/domains/pointing-your-domain/) to run your custom design code with Kinde.
```

Comment:
@clairekinde11 either `to enable your code` or `enable your custom code` - "design code" isn't really something you would say

also `to run your custom code` or `to run your code`